fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabaseRunner.java

```diff
@@ -118,133 +105,75 @@ public FDBDatabaseRunner(@Nonnull FDBDatabase database) {
         this(database, null, null, null);
     }
 
-    /**
-     * Get the database against which functions are run.
-     * @return the database used to run
-     */
+    @Override
```

Comment:
Yeah, I like `{@inheritDoc}` as a general thing. It would be somewhat of an effort to go through all of the things (everywhere in Record Layer) and add it, but I kind of like it.